src/Tasks/Microsoft.NET.Build.Tasks/CheckForDuplicateFrameworkReferences.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.Build.Framework;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    public class CheckForDuplicateFrameworkReferences : TaskBase
+    {
+        [Required]
+        public ITaskItem[] FrameworkReferenceItems { get; set; }
```

Comment:
Why not name this `FrameworkReferences`. Items feels redundant with the `ITaskItem[]` type.

Reply:
It was to resolve the singular/plural discrepancy between `FrameworkReference` as an MSBuild item name and `FrameworkReferences` as a task property :-)

Reply:
Foos="@(Foo)" is common. I don't see issue.